go.mod

```diff
@@ -36,6 +36,7 @@ require (
 	github.com/go-redis/redis/v8 v8.10.0
 	github.com/go-redsync/redsync/v4 v4.4.1
 	github.com/go-sql-driver/mysql v1.6.0
+	github.com/go-test/deep v1.0.8
```

Comment:
would go-cmp work instead? (since we already depend on it and use it in a few places)